### ISSUE TITLE
Implement commentary for Capytains guidelines

### ISSUE THREAD

**Issue by sonofmun** [Capitains/MyCapytain#123]

**Comment by sonofmun** [comment]
If I recall our meeting correctly:

- [ ] In MyCapytain/resources/prototypes/cts/inventory.py, implement a PrototypeCommentary class based on PrototypeText
- [ ] In MyCapytain/resources/collections/cts.py, implement a Commentary class based on the above PrototypeCommentary

**Comment by PonteIneptique** [comment]
I think so yes :)

I'll be available this morning to talk :)

**Comment by PonteIneptique** [comment]
- [ ] Add the about property parsing as well !

**Comment by sonofmun** [comment]
For unit testing:

- [ ] Make sure export works well 
-- create an `__cts__.xml` that we ask to parse and then make sure that all metadata that is there should be there. The abouts, the title, the citation scheme, the description, etc.
-- should export to CTS XML (e.g., http://ctsstage.dh.uni-leipzig.de/api/cts?request=GetCapabilities&urn=urn:cts:latinLit:phi1294)

**Comment by sonofmun** [comment]
Use cases to test:

- [ ] As an API user, I am requesting an inventory. An item of this inventory is a commentary, I want to know information about it
- [ ] As a capitains data provider, I have a set of data where I have commentary. I want metadata to be correctly interpretated by MyCapytain
- [ ] As a Capitains data provider, I want to make sure that my commentaries are correctly displayed through the CTS API

**Comment by sonofmun** [comment]
Is the `<ti:about>` element the only difference between the elements in a `<ti:commentary>` and `<ti:translation>`? I need to set up a commentary element for the unit tests.

**Comment by PonteIneptique** [comment]
Yes, it's the only difference :)

**Comment by sonofmun** [comment]
And `<ti:about>` has the URN that it is about in the `n` attribute? Or as text?

**Comment by PonteIneptique** [comment]
(Putting the issue as reference: https://github.com/Capitains/Capitains.github.io/issues/17 ) 
As `@urn`

**Comment by sonofmun** [comment]
Something like this? Or should the `about` element have some text?
```
<ti:commentary xml:lang='eng' urn='urn:cts:latinLit:phi1294.phi002.perseus-eng3' workUrn='urn:cts:latinLit:phi1294.phi002' xmlns:ti='http://chs.harvard.edu/xmlns/cts'>
  <ti:label xml:lang='eng'>Introduction to Epigrammata</ti:label>
  <ti:description xml:lang='eng'>Introduction to Epigrammata by Someone, 1866, Oxford</ti:description>
  <ti:online></ti:online>
  <ti:about urn="urn:cts:latinLit:phi1294.phi002.perseus-eng2"></ti:about>
</ti:translation>
```

**Comment by PonteIneptique** [comment]
Add commentary object possibility : 
- [ ] https://github.com/Capitains/MyCapytain/blob/master/MyCapytain/resolvers/cts/local.py#L233-L234
- [ ] https://github.com/Capitains/MyCapytain/blob/master/MyCapytain/resolvers/cts/local.py#L305-L310

And test it :)

**Comment by PonteIneptique** [comment]
Fixed by #126
